keps/sig-auth/3766-referential-authorization/README.md

```diff
@@ -558,52 +568,8 @@ moving ReferenceGrant to the new API group, just notes to save discussion time.
   specific `To` Kinds.
 * Corollary for future work, define how controllers interact. Is it a problem if
   multiple controllers consume the same ReferenceGrant?
-* Status design is still pending, but it's currently expected that controllers
-  will indicate status on the _referring_ resources, not on ReferenceGrant itself.
-<<[UNRESOLVED do we need status? ]>>
-Original Thread: https://github.com/kubernetes/enhancements/pull/3767#discussion_r1084670421
-
-We want to be able to represent the following, in descending order of 
-importance: 
-
-1. Communicate that the ReferenceGrant is actively being used
-2. Communicate which controllers are using this ReferenceGrant
-3. Communicate how many times it's been used with sufficient granularity that I
-   can see the effects of my changes (if I remove this reference, am I removing
-   a dependency on this ReferenceGrant?)
-
-We could introduce a status structure that allowed each implementing controller
-to write 1 entry:
-
-```yaml
-status:
-  referencesAllowed:
-  - controllerName: gateway.example.com
-    numReferences: 1
-```
-
-@thockin responded with:
-
-If we think that the cardinality of controllers is low, we can put it in status.
-The downsides are:
-
-1. Could frequently require retries because of optimistic concurrency failures
-   (I'm trying to increment my count, but so is everyone else)
-1. If we're wrong about cardinality, there's not an easy way out
-1. Lots of writes to a resource that will be watched fairly often (every
-   controller which needs refs will watch all refgrants)
-1. We need .status.
-1. If we instead put that into a ReferenceGrantUse resource (just a tuple of
-   controller-name and count), then we only have optimistic concurrency
-   problems with ourselves, we have ~infinite cardinality, nobody will be
-   watching them, and RefGrant doesn't need .status.
-
-Downsides:
-
-1. It's another new resource
-1. It's a new pattern, untested in other places.
-
-<<[/UNRESOLVED]>>
+* Similar to the Gateway API version of ReferenceGrant, controllers will
```

Comment:
Can we (un)officially "reserve" some status sub-field (like `allowedReferenceGrants`), such that one could optimistically query these referring resources' status in some generic way?